audio/mididrv.h

```diff
@@ -213,6 +213,9 @@ class MidiDriver : public MidiDriver_BASE {
 	/** Close the midi driver. */
 	virtual void close() = 0;
 
+	/** Set global music volume */
+	virtual void setVolume(int volume) { }
```

Comment:
This is hides overloaded functions at engines/sci/sound/drivers/amigamac.cpp, engines/parallaction/adlib.cpp, engines/agos/drivers/accolade/adlib.cpp, /engines/queen/midiadlib.h, engines/sherlock/scalpel/drivers/adlib.cpp which define it as setVolume(byte)